third_party/terraform/website/docs/r/compute_instance_group_manager.html.markdown

```diff
@@ -93,7 +93,7 @@ The following arguments are supported:
     which all new instances will be created. This field is only present in the
     `google` provider.
 
-* `version` - (Optional, [Beta](https://terraform.io/docs/providers/google/provider_versions.html)) Application versions managed by this instance group. Each
+* `version` - (Optional for GA, Required for [Beta](https://terraform.io/docs/providers/google/provider_versions.html)) Application versions managed by this instance group. Each
```

Comment:
Our changes in `2.0.0` made `version` `Required` in `google-beta`, but actually removed it from `google` altogether; there, `instance_template` is `Required` while it's not present in `google-beta` at all (and has never been I don't think).

Reply:
👍